apps/codeforafrica/src/lib/api.netlify-cms/utils.js

```diff
@@ -0,0 +1,29 @@
+import fs from "fs";
+import { join } from "path";
+
+import matter from "gray-matter";
+import { marked } from "marked";
+
+export const getCollectionSlugs = (collectionDir) => {
```

Comment:
Since we use `function getCollectionSlugs`, lets keep using that. Or is there a way to mix it up?